Donation-platform/chosen_v1.8.7/composer.json

```diff
@@ -0,0 +1,36 @@
+{
```

Comment:
what are these files

Reply:
This is an external library I added to help with the UI.

Reply:
Do you need all these files? You probably just need to include the js file in the html page. Also are these being used at all? I didn't find any usage.

Reply:
You're right, I will do away with them since I am getting the library from cdn.